.github/workflows/upgrade-python-requirements.yml

```diff
@@ -0,0 +1,28 @@
+name: Upgrade Requirements
+
+on:
+  schedule:
+    # will start the job at 13:15 UTC on 1st of every month
+    - cron: "15 13 1 * *"
+  workflow_dispatch:
+    inputs:
+      branch:
+        description: "Target branch to create requirements PR against"
+        required: true
+        default: '$default-branch'
+
+jobs:
+
+  call-upgrade-python-requirements-workflow:
+     uses: edx/.github/.github/workflows/upgrade-python-requirements.yml@master
```

Comment:
Please add a comment here to mention which script it invoke.

Reply:
The Job **call-upgrade-python-requirements-workflow** is invoke the following script **upgrade-python-requirements.yml**. Please follow the attached link to review the script https://github.com/edx/.github/blob/master/.github/workflows/upgrade-python-requirements.yml#L62-L65